#!/usr/bin/env python
PACKAGE = "waypoint_patroller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("low_battery", int_t, 0, "low battery treshold", 35, 25, 95)
gen.add("very_low_battery", int_t, 0, "panic treshold. robot should callfor help if battery drops below this level", 20, 10, 30)
gen.add("charged_battery", int_t, 0, "undocking treshold", 98, 40, 100)


exit(gen.generate(PACKAGE, "waypoint_patroller", "BatteryTresholds"))